### ISSUE TITLE
Did you retrain the SoTA models for comparison?

### ISSUE THREAD

**Issue by ChrisChen1023** [lyndonzheng/TFill#16]
Hi authors,

Thanks for your excellent work, well done!

I am wondering, did you retrain the SoTA models for comparison? Or you just use the released pre-train models and test with your own test images and makes for testing?

Many thanks for your help! Look forward to your reply!

Best wishes

Chr1s

**Comment by lyndonzheng** [comment]
Hi @ChrisChen1023 Thanks for your interesting. I just used the pre-trained models on their websites for the fair comparisons. Due to the different masks and images, it is hard to get the fair comparison using only the reported numbers from the papers.
